timefold-solver-python-core/src/main/python/_jpype_type_conversions.py

```diff
@@ -184,6 +189,27 @@ def test(self, argument1, argument2, argument3, argument4, argument5):
         return self.delegate(argument1, argument2, argument3, argument4, argument5)
 
 
+def to_python_score(score) -> 'Score':
+    if isinstance(score, _timefold_java_interop._java_score_mapping_dict['SimpleScore']):
+        return _timefold_java_interop._python_score_mapping_dict['SimpleScore'](score.score(),
+                                                                                init_score=score.initScore())
+    elif isinstance(score, _timefold_java_interop._java_score_mapping_dict['HardSoftScore']):
+        return _timefold_java_interop._python_score_mapping_dict['HardSoftScore'](score.hardScore(),
```

Comment:
Should we add the key `hard_score=score.hardScore()` as we did for the init score?

Reply:
No; init_score is kw_only and is optional; hard_score/soft_score etc. are all required and can be specified as keyword or positional.